go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/iotexproject/go-fsm v1.0.0
-	github.com/iotexproject/go-p2p v0.3.0
+	github.com/iotexproject/go-p2p v0.3.2
```

Comment:
in 0.3.2, DHT is set to server mode, which fixed Neighbor() contains no other peers